src/xcode/ENA/ENA/Source/RootCoordinator.swift

```diff
@@ -277,6 +277,7 @@ class RootCoordinator: NSObject, RequiresAppDependencies, UITabBarControllerDele
 		diaryCoordinator.viewController.tabBarItem = diaryTabBarItem
 
 		tabBarController.tabBar.tintColor = .enaColor(for: .tint)
+		tabBarController.tabBar.unselectedItemTintColor = .enaColor(for: .textPrimary2)
```

Comment:
I think you've used two incompatible approaches here. You could either add more images (as you did) and set the rendering mode to original, or keep the template images and add high contrast colors in the color asset catalogue and set both tint colors as you did here. I think extending the colors in the asset catalogue would be preferable as it is more flexible (and it would add support for high contrast mode in other places where we use the colors)